### DISCUSSION TITLE
Redirect to support `with`

### DISCUSSION THREAD

**Discussion by trovster** [livewire/livewire#3068] [7. Bugs (archived)]
**Is your feature request related to a problem? Please describe.**
I want to use the same redirect API as Laravel, so I want to use `redirect()->to()-with($data)` but the `with` method isn't supported.

**Describe the solution you'd like**
Support the `with` method instead of the separate `flash()` messages.

**Comment by squishythejellyfish** [comment]
👋 Oh Hi! I'm Squishy, the friendly jellyfish that manages Livewire issues.

I see this issue has been closed.

Here in the Livewire repo, we have an "issues can be closed guilt-free and without explanation" policy.

If for ANY reason you think this issue hasn't been resolved, PLEASE feel empowered to re-open it.

Re-opening actually helps us track which issues are a priority.

Reply "REOPEN" to this comment and we'll happily re-open it for you!

(More info on this philosophy here: https://twitter.com/calebporzio/status/1321864801295978497)

**Comment by Mane-Olawale** [comment]
I was stuck with this 15 minites ago, This was what i later used
```php
        request()->session()->flash(
            'success',
            'Account infomation submited successfully. Wait for the approval of your account.'
        );
        return redirect()->route('home');
```

**Comment by zepfietje** [comment]
This is still a relevant issue / feature request.

**Comment by zepfietje** [comment]
REOPEN

**Comment by ariaieboy** [comment]
https://github.com/livewire/livewire/pull/3572
